src/components/ButtonPanel.js

```diff
@@ -1,38 +1,46 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPanel = () => (
-  <div>
+const ButtonPanel = ({ clickHandler }) => {
+  const renderButton = (i) => <Button name={i} handleClick={clickHandler} />;
```

Comment:
- [ ] it implements `handleClick` function ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png)). The `handleClick` function should be implemented here.